pom.xml

```diff
@@ -5,7 +5,7 @@
   <parent>
     <groupId>org.jenkins-ci.plugins</groupId>
     <artifactId>plugin</artifactId>
-    <version>2.36</version>
+    <version>2.37</version>
```

Comment:
Possible to go to 3.0 while still using an ancient Jenkins 1.x baseline, but not officially supported.

Note that to test against 2.89.x in `Jenkinsfile` you _do_ need to go to 3.x.